src/workloads/scale/ScanWithLongLived.yml

```diff
@@ -0,0 +1,82 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Engines
+
```

Comment:
Consider adding a `Description` here with a link to the ticket or the doc.

Reply:
baf9639